database/migrations/20190723105203-create-user.js

```diff
@@ -45,7 +45,7 @@ module.exports = {
       allowNull: false,
       type: Sequelize.DATE,
       defaultValue: Sequelize.literal('CURRENT_TIMESTAMP'),
-    }
+    },
   }),
-  down: (queryInterface, Sequelize) => queryInterface.dropTable('Users')
+  down: (queryInterface, Sequelize) => queryInterface.dropTable('Users'),
```

Comment:
Please enforce trailing commas for only multiline array and object statements. It doesn't apply here

Reply:
ESLint flags it when I do that.